engine/src/main/java/org/terasology/rendering/dag/AbstractNode.java

```diff
@@ -15,25 +15,55 @@
  */
 package org.terasology.rendering.dag;
 
-import com.google.common.collect.Lists;
-import com.google.common.collect.Range;
+import com.google.common.collect.Maps;
 import com.google.common.collect.Sets;
-
-import java.util.List;
+import java.util.Map;
 import java.util.Set;
+import org.slf4j.Logger;
+import org.slf4j.LoggerFactory;
+import org.terasology.assets.ResourceUrn;
+import org.terasology.rendering.opengl.BaseFBOsManager;
+import org.terasology.rendering.opengl.FBOConfig;
 
 /**
  * TODO: Add javadocs
  */
 public abstract class AbstractNode implements Node {
+    protected static final Logger logger = LoggerFactory.getLogger(AbstractNode.class);
+
     private Set<StateChange> desiredStateChanges;
+    private Map<ResourceUrn, BaseFBOsManager> fboUsages;
```

Comment:
I'm finding this variable very unclear. I keep confusing it with an usage counter.

I seem to understand this is used only for disposal. How about we call it disposalStore?